packages/ruby/lib/readme/metrics.rb

```diff
@@ -121,34 +109,36 @@ def validate_options(options)
         raise Errors::ConfigurationError, Errors::BUFFER_LENGTH_ERROR
       end
 
-      if options[:development] && !is_a_boolean?(options[:development])
+      if options[:development] && !a_boolean?(options[:development])
         raise Errors::ConfigurationError, Errors::DEVELOPMENT_ERROR
       end
 
-      if options[:logger] && has_logger_inferface?(options[:logger])
+      if options[:logger] && logger_inferface?(options[:logger])
         raise Errors::ConfigurationError, Errors::LOGGER_ERROR
       end
     end
 
-    def has_logger_inferface?(logger)
-      [
-        :unknown,
-        :fatal,
-        :error,
-        :warn,
-        :info,
-        :debug
+    def logger_inferface?(logger)
+      %i[
+        unknown
+        fatal
+        error
+        warn
+        info
+        debug
       ].any? { |message| !logger.respond_to? message }
     end
 
-    def is_a_boolean?(arg)
-      arg == true || arg == false
+    def a_boolean?(arg)
+      [true, false].include?(arg)
     end
 
+    # rubocop:disable Style/InverseMethods
     def user_info_valid?(user_info)
       !user_info.nil? &&
         !user_info.values.any?(&:nil?) &&
-        user_info.has_key?(:api_key) || user_info.has_key?(:id)
+        user_info.key?(:api_key) || user_info.key?(:id)
```

Comment:
Removing `has_` from these was from the recommendation of the [Style/PreferredHashMethods](https://docs.rubocop.org/rubocop/cops_style.html#stylepreferredhashmethods) rule that prefers using `key` and `value` on `Hash` objects instead of `has_key` and `has_value`.